plugins/TrollManagement/TrollManagementTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * @author David Barbier<david.barbier@vanillaforums.com>
+ * @copyright 2009-2021 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace VanillaTests\TrollManagement;
+
+use UserModel;
+use RoleModel;
+use VanillaTests\Models\UserModelTest;
+use VanillaTests\SetupTraitsTrait;
+
+/**
+ * Class TrollManagementTest
+ */
+class TrollManagementTest extends UserModelTest {
+    use SetupTraitsTrait;
+
+    /** @var UserModel */
+    protected $userModel;
+
+    /**
+     * @inheritDoc
+     */
+    public function setUp(): void {
+        parent::setUp();
+    }
```

Comment:
What is this doing?

Reply:
Previously I had dummy user accounts created there but as the Troll plugin needs to be configured prior to the user creation, I moved it. At the moment, this is effectively doing nothing much. Removing it.